lib/model/query/entities.js

```diff
@@ -356,8 +367,16 @@ const _updateEntity = (dataset, entityData, submissionId, submissionDef, submiss
 
 // Used by _updateVerison to figure out the intended base version in Central
 // based on the branchId, trunkVersion, and baseVersion in the submission
-const _computeBaseVersion = (eventId, dataset, clientEntity, submissionDef, forceOutOfOrderProcessing = false) => async ({ Entities }) => {
-  if (!clientEntity.def.branchId) {
+const _computeBaseVersion = (eventId, dataset, clientEntity, submissionDef, forceOutOfOrderProcessing = false, createSubAsUpdate = false) => async ({ Entities }) => {
+  if (clientEntity.def.baseVersion == null && createSubAsUpdate) {
```

Comment:
> If it doesn't forbid it, then I think we should go ahead and apply the submission.

To be more specific, in that case, I think we would just ignore the `baseVersion` specified in the submission.

Reply:
True, I can't think of a scenario where `baseVersion` would be specified and it would get all the way to this point. I don't remember why I added it here. I can take it out of the condition.